src/parser/Expression.ts

```diff
@@ -346,6 +342,49 @@ export class FunctionExpression extends Expression implements TypedefProvider {
         return functionType;
     }
 
+    setReturnType() {
```

Comment:
Could use a doc block

Reply:
the inner description is fine for me. plus I'm converting this to a private method, so the doc block matters even less.